forge/test/mlir/test_ops_tf.py

```diff
@@ -162,12 +163,9 @@ def call(self, x):
     inputs = [tf.random.uniform((1, 128, 128, 3))]
 
     framework_model = DualConv2d()
-    fw_out = to_pt_tensors(framework_model(*inputs))
-
     compiled_model = forge.compile(framework_model, sample_inputs=inputs)
-    co_out = compiled_model(*inputs)
-    co_out = [co.to("cpu").to(fw_out[0].dtype) for co in co_out]
-    assert compare_tensor_to_golden("dual_conv2d", fw_out[0], co_out[0].reshape(fw_out[0].shape))
+
+    verify(inputs=inputs, compiled_model=compiled_model, framework_model=framework_model)
```

Comment:
Minor comment, do we need to run these as named attributes? Can we simplify it like this:
from 
```py
verify(inputs=inputs, compiled_model=compiled_model, framework_model=framework_model)
```
to:
```py
verify(inputs, compiled_model, framework_model)
```

It looks a bit more cleaner, as these attributes are required for every verification. Therefore, we don't need to explicit mention them like so. For other cases, if we expand verify with some "non-common" attributes, we can use named attributes to provide some additional arguments, while not specifying others by order :))